pom.xml

```diff
@@ -82,9 +82,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>mrj0</groupId>
+            <groupId>black.ninia</groupId>
             <artifactId>jep</artifactId>
-            <version>3.5.2</version>
```

Comment:
Since this is a major component might be good to add the change to the release notes. Not required just a thought.